Formula/navi.rb

```diff
@@ -13,6 +13,7 @@ class Navi < Formula
 
   depends_on "rust" => :build
   depends_on "fzf"
+  depends_on "git"
```

Comment:
Is system git not sufficient?

Reply:
Sorry, what does `system git` means exactly?
`navi` now shell outs and calls `git ...`
I just want to make sure that `git` is available when running `navi`
If `git` is guaranteed to be available even without this line, then this line can be removed
Is that the case?

Reply:
You can assume git is available as Homebrew uses git itself to update, unless you specifically require a recent version of git.